layout/_macro/post.swig

```diff
@@ -283,7 +283,7 @@
     <div class="post-body{% if theme.han %} han-init-context{% endif %}{% if post.direction && post.direction.toLowerCase() === 'rtl' %} rtl{% endif %}" itemprop="articleBody">
 
       {# Gallery support #}
-      {% if post.photos and post.photos.length %}
+      {% if post.photos and post.photos.length and (theme.photos.display_in_post or is_index) %}
```

Comment:
but u need to add cover photo in post manually?

Reply:
yes